articles/server-platforms/rails/00-introduction.md

```diff
@@ -0,0 +1,78 @@
+## Ruby on Rails - Introduction
```

Comment:
Please change this to "Introduction", as is it does not display like the rest:

![image](https://cloud.githubusercontent.com/assets/11715799/16945791/3ae67ab6-4daf-11e6-81e7-3e680238cb8d.png)